scale_build/config.py

```
@@ -0,0 +1,21 @@
+import os
+import time
+
+from datetime import datetime
+
+
+BUILD_TIME = int(time.time())
+BUILD_TIME_OBJ = datetime.fromtimestamp(BUILD_TIME)
+BUILDER_DIR = os.getenv('BUILDER_DIR', './')
+BRANCH_OVERRIDES = {k[:-(len('_OVERRIDE'))]: v for k, v in os.environ.items() if k.endswith('_OVERRIDE')}
```

Should we fail if an unknown package is specified here?

---

I don't think so, we won't be consuming it.

---

I think we should fail if there is an unconsumed environment variable (e.g. due to a typo)

---

Any unconsumed env variable ?

---

There could be more then just this example

---

On all that get into `BRANCH_OVERRIDES`. E.g. `MILDLEWARE_OVERRIDE=xxx`